File: Kanishk-Tyagi/PrefixSum/findHighestAltitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int current_altitude=0;
        int highest_altitude=0;
        for(int i=0;i<gain.size();i++){
            current_altitude += gain[i];
            highest_altitude=max(current_altitude, highest_altitude);
        }
        return highest_altitude;
    }
};

File: Anushka-Pachauri/PrefixSum/FindHighestAltitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int maxAltitude=0;
        for(int i=1;i<gain.length;i++){
            gain[i]+=gain[i-1];
        }
        for(int i=0;i<gain.length;i++){
            maxAltitude=Math.max(maxAltitude,gain[i]);
        }
        return maxAltitude;
    }
}